src/briefcase/macos.py

```diff
@@ -58,6 +62,14 @@ def launcher_script_location(self):
         return macos_dir
 
     def build_app(self):
+        print("Building DMG file...")
+        dmg_name = self.formal_name + '.dmg'
+        dmg_path = os.path.join(os.path.abspath(self.dir), dmg_name)
+        settings = {'files': [self.app_location],
+                    'symlinks': {'Applications': '/Applications'}}
+        dmgbuild.build_dmg(filename=dmg_path,
+                           volume_name=self.formal_name, settings=settings)
```

Comment:
At column 86: (E261) at least two spaces before inline comment

Reply:
What inline comment? could someone look at this?

Reply:
Looks like it's talking about the comment on line 30.